forte/pipeline.py

```diff
@@ -176,6 +186,8 @@ def initialize_processors(self):
             try:
                 processor.assign_manager(self._proc_mgr)
                 processor.initialize(self.resource, config)
+                if self._check_type_consistency:
+                    processor.enforce_consistency()
```

Comment:
how about the check for evaluator?

Reply:
From my understanding, Evaluator inherets PipelineComponent and its also part of self.components here. Would it be also initialized and set _check_type_consistency here?

Reply:
I see. The code wrote "processor" so I thought this is just for "processor", but here it actually mean "evaluator"